setup.py

```diff
@@ -179,6 +179,9 @@ def get_exts_for(name):
                 "pyddprofile = ddtrace.profiling.__main__:main",
             ],
             "pytest11": ["ddtrace = ddtrace.contrib.pytest.plugin"],
+            "gevent.plugins.monkey.did_patch_all": [
```

Comment:
Today I learned this existed! We should probably do something similar in the tracer (in a follow up)

Reply:
Also I wonder if we can offer something similar to solve the issue @zhammer had with https://github.com/DataDog/dd-trace-py/issues/1344